DPG/Tasks/CMakeLists.txt

```diff
@@ -10,50 +10,54 @@
+        SOURCES qaEventTrackLite.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2::ReconstructionDataFormats
+        COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(qa-efficiency
-                    SOURCES qaEfficiency.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
+        SOURCES qaEfficiency.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+        COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(qa-impact-parameter
-                    SOURCES qaImpPar.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework
-                                          O2Physics::AnalysisCore
-                                          O2::ReconstructionDataFormats
-                                          O2::DetectorsBase
-                                          O2::DetectorsCommonDataFormats
-                                          O2::DetectorsVertexing
-                    COMPONENT_NAME Analysis)
+        SOURCES qaImpPar.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework
+        O2Physics::AnalysisCore
+        O2::ReconstructionDataFormats
+        O2::DetectorsBase
+        O2::DetectorsCommonDataFormats
+        O2::DetectorsVertexing
+        COMPONENT_NAME Analysis)
 
 # PID QA
 
 ## TOF
 o2physics_add_dpl_workflow(pid-tof-qa
-                    SOURCES qaPIDTOF.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
+        SOURCES qaPIDTOF.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
+        COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(pid-tof-qa-mc
-                    SOURCES qaPIDTOFMC.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
+        SOURCES qaPIDTOFMC.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
+        COMPONENT_NAME Analysis)
 
 ## TPC
 o2physics_add_dpl_workflow(pid-tpc-qa
-                    SOURCES qaPIDTPC.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
+        SOURCES qaPIDTPC.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
+        COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(pid-tpc-qa-mc
-                    SOURCES qaPIDTPCMC.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
+        SOURCES qaPIDTPCMC.cxx
+        PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2Physics::AnalysisCore
+        COMPONENT_NAME Analysis)
+##FDD
+  o2physics_add_dpl_workflow(fdd-qa
+          SOURCES fddQA.cxx
```

Comment:
you could consider renaming the task to ´qaFDD´ to be consistent with the other DPG-QA task names.